connection.go

```diff
@@ -251,12 +254,18 @@ func (ch *Channel) newConnection(conn net.Conn, initialState connectionState, ev
 	c.log = log
 	c.inbound.onRemoved = c.checkExchanges
 	c.outbound.onRemoved = c.checkExchanges
+	c.inbound.onAdded = c.onExchangeAdded
```

Comment:
There is not much difference here. If we have it, it is easy for us to debug and trace the event.